Shared/Camera/Pov.cs

```diff
@@ -164,23 +175,43 @@ private void OnDestroy()
 
         private void SetVisibility(ChaControl chara)
         {
-            if (chara != null) chara.fileStatus.visibleHeadAlways = true;
+            if (chara == null) return;
+            
+            chara.fileStatus.visibleHeadAlways = true;
+
+            var hair = chara.objHeadBone.transform.Find("ct_head/N_tonn_face/N_cf_haed");
+
+            if (hair != null)
+                hair.gameObject.SetActive(true);
```

Comment:
It may be faster to check if it's active first if hair is likely to be already active.